coalib/bears/Bear.py

```diff
@@ -1,4 +1,9 @@
+import cProfile
 import inspect
+import io
```

Comment:
This file contains unused source code.

*Origin: PyUnusedCodeBear, Section: `flakes`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpykm87f6n/coalib/bears/Bear.py
+++ b/tmp/tmpykm87f6n/coalib/bears/Bear.py
@@ -1,6 +1,5 @@
 import cProfile
 import inspect
-import io
 import itertools
 import pstats
 import sys
```